pkg/network/protocols/events/batch_offsets.go

```diff
@@ -0,0 +1,93 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux_bpf
+// +build linux_bpf
+
+package events
+
+import (
+	"sync"
+)
+
+// offsetManager is responsible for keeping track of which chunks of data we
+// have consumed from each batch object
+type offsetManager struct {
```

Comment:
consider renaming (maybe `batchOffsetManager`) to make a difference between offsetGuessing
it might confuse the average reader

Reply:
I didn't see any change, let me know if you disagree with the change

Reply:
given that there is a good amount of documentation here and that the code you're referring to belongs to a different package, I don't think this is necessary